blazor/ClientSideExample/ClientSideExample/Views/GreetingView.razor.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using ClientSideExample.ViewModels;
+using ReactiveUI;
+
+namespace ClientSideExample.Views
+{
+    public partial class GreetingView : IViewFor<GreetingViewModel>
+    {
+        public GreetingView()
+        {
+            ViewModel = new GreetingViewModel();
+        }
+
+        public GreetingViewModel ViewModel { get; set; }
+        object IViewFor.ViewModel 
+        {
+            get => ViewModel;
+            set => ViewModel = (GreetingViewModel)value;
+        }
+
+        public void Clear()
```

Comment:
Return Task here and write `Execute().ToTask()` so that Clear is testable

Reply:
Done.